appservice/src/SiteWrapper.ts

```diff
@@ -183,14 +183,24 @@ export class SiteWrapper {
 
     public async deploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, configurationSectionName: string, confirmDeployment: boolean = true): Promise<void> {
         const config: SiteConfigResource = await this.getSiteConfig(client);
+        const kuduClient: KuduClient = await this.getKuduClient(client);
+        const deploymentId: string = 'latest';
+        let previousDeploymentId: string;
+        try {
+            previousDeploymentId = (await kuduClient.deployment.getResult(deploymentId)).id;
+            // current active deployment used to verify new deployment was successful
+        } catch (error) {
```

Comment:
Perhaps you should check that the error is what you're expecting for this scenario?

Reply:
@StephenWeatherford I just talked to Nathan offline and I think we should swallow all errors here. No matter if or how it fails, we should still move on and try to deploy IMO

Reply:
Discussing with Eric, it may not be best to check for the specific error message we're expecting.

If the message ever changes or is localized, this will break the deployment.
If we get an error at this point of the deployment, this shouldn't break the deployment.  This is only for validation and we account for undefined ids.

If there is an error, it's better for it to be caught in the specific deployment command rather than during the check for a latest id.